htdocs/compta/facture/card.php

```diff
@@ -5129,6 +5134,11 @@ function js_recalculate_revenuestamp(){
 
 		// Remainder to pay back Multicurrency
 		if ($object->multicurrency_code != $conf->currency || $object->multicurrency_tx != 1) {
+			$multicurrency_payment = $invoice->getSommePaiement(1);
```

Comment:
There is no reason to make a lot of subrequest to get the remain to pay in multicurrency.
We already have the remain to pay and the rate. 
So remain to pay in currency should be the rate * remain to pay in main currency.

Reply:
Due to the forex conversion and the rounding, it happens very often that the factual remainder is 0 in the original currency but 0.01 in the main currency. The below shows an example after I applied the new coding, otherwise it will results in 0.01 in both currencies. To supplement, in my case 1 local dollar = 0.75 USD. You can double check, the billed amount rounds up and resulted in the excess of 0.01 local currency.

![image](https://user-images.githubusercontent.com/3304600/131666491-231a58e8-0a88-43d6-a794-5bceb9beef48.png)